src/pkg/repository/loadtest/repository_load_test.go

```diff
@@ -384,38 +384,52 @@ func normalizeCategorySet(t *testing.T, cats map[string]struct{}) []string {
 	return sl
 }
 
+/* ================================================
+*   A note on load test setup:
+*   Even though most of the code here is boiler-
+*   plate and could be easily compressed into a
+*   test matrix, we want to keep the suites separte
+*   to maximize parallelism.  Due to how testify's
+*   suites work, we can only run in parallel at the
+*   level of the suite, not within each test.
+*  ================================================ */
+
 // ------------------------------------------------------------------------------------------------
 // Exchange
 // ------------------------------------------------------------------------------------------------
 
 // multiple users
 
-type RepositoryLoadTestExchangeSuite struct {
-	suite.Suite
+type LoadExchangeSuite struct {
+	tester.Suite
 	ctx            context.Context
 	repo           repository.Repository
 	acct           account.Account //lint:ignore U1000 future test use
 	st             storage.Storage //lint:ignore U1000 future test use
 	usersUnderTest []string
 }
 
-func TestRepositoryLoadTestExchangeSuite(t *testing.T) {
-	tester.RunOnAny(t, tester.CorsoLoadTests)
-	suite.Run(t, new(RepositoryLoadTestExchangeSuite))
+func TestLoadExchangeSuite(t *testing.T) {
+	suite.Run(t, &LoadExchangeSuite{
+		Suite: tester.NewLoadSuite(
+			t,
+			[][]string{tester.AWSStorageCredEnvs, tester.M365AcctCredEnvs},
```

Comment:
should we just make this part of the default check for the load suite in tester package?

Reply:
not necessarily.  It's a nominal addition, and we may want this granular control for local testing or if/when we introduce non-microsoft platforms.